features/step_definitions/assertion_steps.rb

```diff
@@ -12,6 +12,10 @@
     expect(page).to have_no_content content
 end
 
+Then("I should be on the videocall page") do
```

Comment:
You are not routing to this page anymore

Reply:
Done